README.md

```diff
@@ -42,10 +42,7 @@ For notarization, you need the following things:
 1. Xcode 10 or later installed on your Mac.
 2. An [Apple Developer](https://developer.apple.com/) account.
 3. [An app-specific password for your ADC account’s Apple ID](https://support.apple.com/HT204397).
-4. Your app may need to be signed with harded-runtime and the following entitlements:
-    1. com.apple.security.cs.allow-jit
-    1. com.apple.security.cs.allow-unsigned-executable-memory
-    1. com.apple.security.cs.allow-dyld-environment-variables
+4. Your app may need to be signed with hardened-runtime and [list of entitlements](https://developer.apple.com/documentation/bundleresources/entitlements)
```

Comment:
You definitely need at least `com.apple.security.cs.allow-unsigned-executable-memory` and possibly `com.apple.security.cs.allow-jit` (Though I remain unconvinced: https://github.com/electron/electron-osx-sign/issues/188#issuecomment-483778579) so not mentioning those would be unkind.

The `allow-jit` one at least does no harm (especially if it's superseded by/included in `allow-unsigned-executable-memory`), so lets keep it at those two.